### ISSUE TITLE
Replace Google Form with using github issues for new data portals

### ISSUE THREAD

**Issue by rufuspollock** [okfn/dataportals.org#90]
*This is a suggestion -- discussion welcome!*

What: replace Google Form with using github issues for new data portals

Why?

* Github issues would be public - and anyone could see and act on them
* Github issues can be discussed
* They integrate with our tracker here - one less thing to check
* Description are in markdown for us automatically!

Why not?

* Google form is more structured
  * I think we can enforce a sample structure for a github issue - https://help.github.com/articles/creating-an-issue-template-for-your-repository/
* it's a change

/cc @Stephen-Gates @danfowler thoughts?

**Comment by todrobbins** [comment]
I like this idea and I think issue templating will cover most use cases.

**Comment by rufuspollock** [comment]
@todrobbins great - would you be up for having a first pass at mocking up an issue template?

**Comment by todrobbins** [comment]
@rufuspollock sure. I'll post my draft here.

**Comment by todrobbins** [comment]
Here we go @rufuspollock:

```
`* Required`

# Key Information

## Title *
<!-- Full title of the portal, e.g. "Data.Gov.UK - Official UK Government Portal" -->

## URL *
<!-- URL to home page of the portal -->

## Manager
<!-- Name/title of the organization that set up and manages the portal -->

## Description *
<!-- Description and any additional notes about the portal, for example special licensing conditions, comments on the contents, or anything that doesn't fit in the questions below but is relevant -->

# Additional Information
## Place *
<!-- Short text describing the place that the portal is relate to (i.e. is supposed to cover). Should be sufficient to look it up on a map. For example, "London, UK" or "Vancouver, Canada". If no specific location just put "International" -->

## Country
<!-- Country the portal covers. 2-digit ISO-code if possible. Note we allow regional answers if that is appropriate (e.g. EU) -->

## License *
<!-- Default license for the data in the data portal. You will be able to provide more information in the next question. Note "license" is used loosely to cover the general terms of use under which data is made available. -->

## License details
<!-- Provide here any additional details related to your previous answer. For example, the URL to the license on the site, or excerpts from the relevant terms of use. -->

## Type *
<!-- Is the portal operated by (and for) government (including IGOs such as World Bank or EC), research (academia), (other) institutional, or community? -->

## Launched
<!-- Date the portal was launched if known. If you are not sure of month and day just choose Jan 1st. Further info (for example a blog post URL about the launch) can go in the the description.

Date formate: mm/dd/yyyy
-->

## Portal Software
<!-- What software platform does the portal use -->

## Portal Software details
<!-- If you answered 'Other' or 'Custom/in-house' implementation to the question above, please feel free to give more details here. If there is a platform we've missed, we'll add it to the list. -->

## API Type *
<!-- Does the portal have any kind of API (structured metadata about the data that can be downloaded)? If multiple options apply, please choose the most extensive/advanced API. -->

## API Endpoint
<!-- URL for accessing the API -->

## Full metadata download
<!-- URL to download entire portal contents, either via an API, or dump provided by the publisher or a 3rd party -->

# Your Information
## Your Name
<!-- So we know who is submitting this and can credit you as appropriate. -->

## Your Email
<!-- We won't share this with anyone else but it enables us to follow up if needed -->

## Your Link
<!-- A link we could use if we need to credit you -->

**By submitting material in the form you agree to:**

A. The terms of use - https://okfn.org/terms-of-use/

B. To place your contribution in the public domain (specifically you "license" any rights you may have under the CCZero - http://creativecommons.org/publicdomain/zero/1.0/ and/or the Open Data Commons Public Domain Dedication and License - http://opendatacommons.org/licenses/pddl/1.0/).

This latter enables us, in turn, to make available the resulting database openly available as per the Open Definition - http://opendefinition.org/od/.
```

Which will render like so with a submitted issue:

---

`* Required`

# Key Information

## Title *


## URL *


## Manager


## Description *


# Additional Information
## Place *


## Country


## License *


## License details


## Type *


## Launched


## Portal Software


## Portal Software details


## API Type *


## API Endpoint


## Full metadata download


# Your Information
## Your Name


## Your Email


## Your Link


**By submitting material in the form you agree to:**

A. The terms of use - https://okfn.org/terms-of-use/

B. To place your contribution in the public domain (specifically you "license" any rights you may have under the CCZero - http://creativecommons.org/publicdomain/zero/1.0/ and/or the Open Data Commons Public Domain Dedication and License - http://opendatacommons.org/licenses/pddl/1.0/).

This latter enables us, in turn, to make available the resulting database openly available as per the Open Definition - http://opendefinition.org/od/.

**Comment by rufuspollock** [comment]
@todrobbins looks good.

Also good to explain for power users that they can contribute via fork and pull on the CSV file -- can we have that as part of the template or not?

**Comment by todrobbins** [comment]
@rufuspollock we can add that comment at the top of the template like so:

```
<!--
Power users can fork and pull the DataPortals.org CSV data file directly:

https://github.com/okfn/dataportals.org/blob/master/data/portals.csv
-->


`* Required`
...
-->
```

**Comment by rufuspollock** [comment]
@todrobbins sounds great.

**Comment by todrobbins** [comment]
We're live!

![screen shot 2017-04-07 at 9 36 46 am](https://cloud.githubusercontent.com/assets/158590/24807548/ddbdc7e6-1b75-11e7-86f9-d5aa3c54ece5.png)

Next step is to replace http://dataportals.org/add with at least a link to create a new issue on Github. I don't think you can embed.

**Comment by todrobbins** [comment]
resolved by #98
